clients/vscode/src/Commands.ts

```diff
@@ -20,6 +20,29 @@ const toogleEnabled: Command = {
   },
 };
 
+const setSuggestionDelay: Command = {
```

Comment:
consider run [prettier](https://prettier.io/playground) for the block

Reply:
Formatted code block with prettier, thank you for comment.